src/patients/labs/Labs.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react'
```

Comment:
Need tests for this component: 

- [ ] test that the table renders with the correct labs
- [ ] test that the content of the table is correct
- [ ] test that when no labs are returned, the warning message appears